examples/kubernetes/1.11/cilium-crio-ds.yaml

```diff
@@ -42,8 +42,6 @@ spec:
                 - "NET_ADMIN"
```

Comment:
Same here, typo in commit message: cri-o 1.11 --> k8s 1.11

Reply:
it's not a typo

Reply:
Resolved in 1.1 backport thread for this change.